src/components/common/WebsiteCard.tsx

```diff
@@ -1,14 +1,6 @@
 import Image from "next/image";
 import TagBar from "./TagBar";
-
-type WebsiteCardProps = {
-  id: string;
-  title: string;
-  tags: string[];
-  primaryAuthor: string;
-  coverImage: string;
-  profile: string;
-};
```

Comment:
Why was this removed?

Reply:
Ah, nvm i just realized it was only moved

Reply:
yeah, I adjust it.